generator/pom.xml

```diff
@@ -98,25 +101,508 @@
                         </goals>
                         <configuration>
                             <sourcePaths>
-                                <sourcePath>${sdk.project.dir}/src/main/resources/schema/artifact-published-event.json
-                                </sourcePath>
+                                <sourcePath>${parent.project.dir}/spec/schemas/artifactpublished.json</sourcePath>
                             </sourcePaths>
+                            <outputDirectory>${sdk.project.dir}/src/main/java</outputDirectory>
                             <targetPackage>dev.cdevents.models.artifact.published</targetPackage>
+                            <includeHashcodeAndEquals>false</includeHashcodeAndEquals>
+                            <includeToString>false</includeToString>
                         </configuration>
                     </execution>
                     <execution>
-                        <id>generate-pipeline-run-finished-from-schema</id>
+                        <id>generate-branch-created-from-schema</id>
                         <phase>generate-sources</phase>
                         <goals>
                             <goal>generate</goal>
                         </goals>
                         <configuration>
                             <sourcePaths>
-                                <sourcePath>
-                                    ${sdk.project.dir}/src/main/resources/schema/pipeline-run-finished-event.json
-                                </sourcePath>
+                                <sourcePath>${parent.project.dir}/spec/schemas/branchcreated.json</sourcePath>
                             </sourcePaths>
+                            <outputDirectory>${sdk.project.dir}/src/main/java</outputDirectory>
```

Comment:
This is a bit long to properly review manually, did you generate it with a script, or do you have a script to verify its correctness?